sklearn/cluster/dbscan_.py

```diff
@@ -50,6 +50,11 @@ def dbscan(X, eps=0.5, min_samples=5, metric='minkowski',
         must be square. X may be a sparse matrix, in which case only "nonzero"
         elements may be considered neighbors for DBSCAN.
 
+    metric_params : dict, optional (default = None)
```

Comment:
I think "optional" is sufficient when the semantics of the default are not clear.